src/template.html

```diff
@@ -22,6 +20,10 @@
       <i class="settings"></i>
     </a>
 
+    <a class="octotree_spin">
+      <img src="https://assets-cdn.github.com/images/spinners/octocat-spinner-32-EAF2F5.gif" />
```

Comment:
I don't think this is a good idea. We can't control it (e.g. what if the URL changes). Should stick with CSS-based spinner.